packages/types/src/cart/common.ts

```diff
@@ -349,7 +366,7 @@ export interface CartLineItemDTO {
   /**
    * The price of the item
    */
-  unit_price: number
+  unit_price: BigNumber
```

Comment:
q: wdyt about external types being bound to string/number and the module handles bignumber internally?

Reply:
+1